MergerLogic/Utils/GpkgUtils.cs

```diff
@@ -369,5 +369,305 @@ public void Vacuum()
 
             this._timeUtils.PrintElapsedTime("Vacuum runtime", ts);
         }
+
+        public void Create(Extent extent,int maxZoom)
+        {
+            SQLiteConnection.CreateFile(this.path);
```

Comment:
add info log for the create function
add debug log for each of the inner "create" methods